src/types/AppNextButtonProps.ts

```diff
@@ -0,0 +1,6 @@
+interface AppNextButtonProps {
```

Comment:
A more generic button props or different name would be nicer imho, since those look to be properties that may apply to anny button, not only the "next" ones :)